src/controllers/comments.controller.js

```diff
@@ -134,23 +124,21 @@ class Comments {
     const getComment = await CommentsDb.findOne({ where: { id: req.params.id } });
     const gottenComent = getComment && getComment.get().id;
     const { id } = req.params;
-    if (!id) {
-      return res.status(400).send({
-        status: 400,
-        message: 'Please provide valid numeric value'
-      });
-    }
+
     if (!gottenComent) {
-      return res.status(200).json({ status: 200, message: 'That comment does not exist' });
+      await util.setSuccess(200, 'That comment does not exist');
+      return util.send(res);
+    }
+    if (!Number(id)) {
+      await util.setError(400, 'Please provide numeric value');
+      return util.send(res);
     }
 
     const { body } = req.body;
-    const updateComment = await commentsService.updateComment(req.params.id, { body });
-    return res.status(200).send({
-      status: 200,
-      message: 'Updation is successfully',
-      updateComment
-    });
+    const commentRevisions = getComment.dataValues.body;
+    const updateComment = await commentsService.updateComment(req.params.id, { body, commentRevisions });
+    await util.setSuccess(200, 'Updation is successfully', updateComment);
```

Comment:
Update is successful.

Reply:
@I have returned update is successfully instead of Updation is successful as You suggested, Thank You